src/samples/UnAuthChildBenefitsClaim/index.tsx

```diff
@@ -31,6 +31,7 @@ import {
 } from '../../components/AppComponents/TimeoutPopup/timeOutUtils';
 import DeleteAnswers from './deleteAnswers';
 import TimeoutPopup from '../../components/AppComponents/TimeoutPopup';
+import toggleNotificationProcess from '../../components/helpers';
```

Comment:
I think this should be "import toggleNotificationProcess from '../../components/helpers/toggleNotificationLanguage';" ?